packages/framework/src/Model/Product/ProductFacade.php

```diff
@@ -141,18 +150,24 @@ public function edit($productId, ProductData $productData)
 
         if (!$product->isMainVariant()) {
             $this->productManualInputPriceFacade->refreshProductManualInputPrices($product, $productData->manualInputPricesByPricingGroupId);
-        } else {
+        }
+
+        if ($product->isMainVariant()) {
             $product->refreshVariants($productData->variants);
         }
+
         $this->refreshProductAccessories($product, $productData->accessories);
         $this->em->flush();
         $this->productSellingDeniedRecalculator->calculateSellingDeniedForProduct($product);
+
         $this->imageFacade->manageImages($product, $productData->images);
         $this->friendlyUrlFacade->saveUrlListFormData('front_product_detail', $product->getId(), $productData->urls);
-        $this->createFriendlyUrlsWhenRenamed($product, $originalNames);
```

Comment:
after this method call was removed, it is not used anymore, I am a little bit concerned about the removal but if we really want to get rid of this behavior, we should remove the unused method

Reply:
well, removed :)